demo/admin/src/pages/blocks/FullWidthImageBlock.tsx

```diff
@@ -9,19 +9,32 @@ const FullWidthImageContentBlock = createOptionalBlock(RichTextBlock, {
     title: <FormattedMessage {...messages.content} />,
 });
 
-export const FullWidthImageBlock = createCompositeBlock({
-    name: "FullWidthImage",
-    displayName: <FormattedMessage id="blocks.fullWidthImage" defaultMessage="Full Width Image" />,
-    category: BlockCategory.Media,
-    blocks: {
-        image: {
-            block: DamImageBlock,
-            title: <FormattedMessage {...messages.image} />,
-            paper: true,
-        },
-        content: {
-            block: FullWidthImageContentBlock,
-            title: <FormattedMessage {...messages.content} />,
+export const FullWidthImageBlock = createCompositeBlock(
+    {
+        name: "FullWidthImage",
+        displayName: <FormattedMessage id="blocks.fullWidthImage" defaultMessage="Full Width Image" />,
+        category: BlockCategory.Media,
+        blocks: {
+            image: {
+                block: DamImageBlock,
+                title: <FormattedMessage {...messages.image} />,
+                paper: true,
+            },
+            content: {
+                block: FullWidthImageContentBlock,
+                title: <FormattedMessage {...messages.content} />,
+            },
         },
     },
-});
+    (block) => ({
+        ...block,
+        extractTextContents: (block) => {
+            const content = block.content?.block ? FullWidthImageContentBlock.extractTextContents?.(block.content ?? {}) ?? [] : [];
```

Comment:
`block.content` should never be undefined. It should be possible to call `FullWidthImageContentBlock.extractTextContents?.(block.content) ?? []`